source/guides/maintenance/cpu-performance.rst

```diff
@@ -193,6 +193,53 @@ group:
    After adding a new group, you must log out and log back in for the new group
    to take effect.
 
+Thermal configuration
+=====================
+
+Better thermal control and performance can be archived by providing platform
+specified configuration to thermald.
+
+`Linux DPTF Extract Utility`_ is a companion tool to thermald, This tool can
+make use of "Intel ® Dynamic Platform and Thermal Framework (`Intel DPTF`_)"
+technology, and convert to the thermal_conf.xml configuration format used
+by thermald.
+
+First, make sure thermald version is equal or above 1.9.1:
+
+.. code:: bash
+
+   thermald --version
+
+Otherwise, you need to add an option :command:`--ignore-default-control` to
+thermald:
+
+.. code:: bash
+
+   # edit thermald.servcie
+   vim /usr/lib/systemd/system/thermald.service
+   # append --ignore-default-control option to ExecStart line, like below
+   # ExecStart=/usr/bin/thermald --no-daemon --dbus-enable --ignore-default-control
```

Comment:
Make Ln 221 a separate example, like so: 
```suggestion
.. code:: bash
   ExecStart=/usr/bin/thermald --no-daemon --dbus-enable --ignore-default-control
```
Assure that this line is **NOT** commented out with a '#'